src/SmartComponents/DriftPage/FilterDropDown/FilterDropDown.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+
+import { compareActions } from '../../modules';
+
+class FilterDropDown extends Component {
+    constructor(props) {
+        super(props);
+        this.onToggle = this.onToggle.bind(this);
+        this.getFilterbyState = this.getFilterByState.bind(this);
+    }
+
+    onToggle() {
+        this.props.toggleDropDown();
+    }
+
+    getFilterByState(event, filter, selected) {
+        let filterData = { filter, selected };
+        this.props.changeFilter(filterData);
+    }
+
+    render() {
+        const dropdownItems = [
+            <DropdownItem key= "none" component="button" onClick={ event =>
+                this.getFilterByState(event, 'none', 'No Filter')
+            }>
+                No Filter
+            </DropdownItem>,
```

Comment:
Can these items be generated programmatically since they share the same structure?